tests/pagination.py

```diff
@@ -75,3 +75,30 @@ def test_paginate_function(app):
         def foo_instance():
             return Cursor([1, 2, 3])
         assert paginate(foo_instance, skip=2)().data == [3]
+
+
+@pytest.mark.parametrize("app", [
+    Stupeflask(__name__),
+    Flask(__name__)
+])
+def test_paginate_header_total_count(app):
+    with app.test_request_context():
+        paginate(Cursor([1, 2, 3]), headers=True)
+        if isinstance(app, Stupeflask):
+            assert request.response_headers["X-Total-Count"] == 3
```

Comment:
I feel that this test is misleading, because we check that the header exist in `request.reponse_headers`, but we don't check that the **response** actually has the headers.

And with the current implementation, the response doesn't have them since it has only been implemented in the `flask_stupe.json.Stupeflask` module, not the base one!

Reply:
Resolving because https://github.com/numberly/flask-stupe/pull/27/files#r237106990 makes sense.